psh/test-control.py

```diff
@@ -0,0 +1,91 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh control codes test
+#
+# Copyright 2023 Phoenix Systems
+# Author: Damian Modzelewski
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+import tools.psh as psh
+
+
+def CC_parser(harness, input: str, output: str):
+    harness.write(input)
+    if not output:
+        harness.expect(psh.PROMPT)
+    else:
+        harness.expect(f"psh: {output} not found", timeout=-1)
+
+
+"""
+CONTROL CODE CHEAT-SHEAT
```

Comment:
Greate idea with control code cheat sheet
```suggestion
CONTROL CODE CHEAT-SHEET
```